src/Microsoft.ML.StandardLearners/FactorizationMachine/FactorizationMachineTrainer.cs

```diff
@@ -509,7 +513,8 @@ internal static CommonOutputs.BinaryClassificationOutput TrainBinary(IHostEnviro
             return new FieldAwareFactorizationMachinePredictionTransformer(Host, model, trainData.Schema, FeatureColumns.Select(x => x.Name).ToArray());
         }
 
-        public FieldAwareFactorizationMachinePredictionTransformer Fit(IDataView input) => Train(input);
+        /// <summary> Trains and returns a <see cref="ITransformer"/>.</summary>
+        public FieldAwareFactorizationMachinePredictionTransformer Fit(IDataView input) => Fit(input);
```

Comment:
>public FieldAwareFactorizationMachinePredictionTransformer Fit(IDataView input) => Fit(input); [](start = 8, length = 94)

Ghostbusters theme playing.
If there's something strange,
In your test runs.
Who you gonna blame?
STACKOVERFLOW!

Reply:
on a serious note, you call yourself here.
change it to `Fit(IDataView input) => Fit(input,null, null)`

---
In reply to: [256597651](https://github.com/dotnet/machinelearning/pull/2528#discussion_r256597651) [](ancestors = 256597651)

Reply:
Ah makes sense! Thank you I thought there was something strange thanks a lot 👍